test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildRestoreTaskTests.cs

```diff
@@ -1136,7 +1136,7 @@ public void MsbuildRestore_WithCPPCliVcxproj_WithProjectReferenceAndWindowsWindo
         [PlatformTheory(Platform.Windows)]
         [InlineData(true)]
         [InlineData(false)]
-        public async Task MsbuildRestore_PackagesConfigDependency_WithHttpSource_Warns(bool useStaticGraphEvaluation)
+        public async Task MsbuildRestore_PackagesConfigDependency_WithHttpSource_DispalyesAnError(bool useStaticGraphEvaluation)
```

Comment:
Typo:

```suggestion
        public async Task MsbuildRestore_PackagesConfigDependency_WithHttpSource_DisplaysAnError(bool useStaticGraphEvaluation)
```

I suggest removing "Displays" altogether though since that implies UI. Errors or ResultsInError. Example:
```suggestion
        public async Task MsbuildRestore_PackagesConfigDependency_WithHttpSource_Errors(bool useStaticGraphEvaluation)
```